tf/service_cluster/container.tf

```diff
@@ -23,9 +23,14 @@ resource "google_container_cluster" "main" {
   ip_allocation_policy {
   }
 
-  node_pool_auto_config {
-    network_tags {
-      tags = var.add_node_pool_network_tags
+  dynamic "node_pool_auto_config" {
```

Comment:
This seems like a bug in the provider?

Reply:
This is either a bug or a known hack on terraform side. I can open an issue.

Reply:
I see you already did it in april.
https://github.com/hashicorp/terraform-provider-google/issues/17914
Should we just add this workaround until a fix is online? Looks like it can take a while